### DISCUSSION TITLE
テキスト中のvarの利用について

### DISCUSSION THREAD

**Discussion by UMA0626** [asciidwango/js-primer#1387] [質問]
以下のページの「変数名に使える名前のルール」セクションで、変数として利用できる文字列の例の部分で「var」が使用されています。
https://jsprimer.net/basic/variables/#variable-name
```js
var $; // OK: $が利用できる
var _title; // OK: _が利用できる
var jquery; // OK: 小文字のアルファベットが利用できる
var TITLE; // OK: 大文字のアルファベットが利用できる
var es2015; // OK: 数字は先頭以外なら利用できる
var 日本語の変数名; // OK: 一部の漢字や日本語も利用できる
```
```js
var 1st; // NG: 数字から始まっている
var 123; // NG: 数字のみで構成されている
```
```js
var var; // NG: `var`は変数宣言のために予約されているので利用できない
var if; // NG: `if`はif文のために予約されているので利用できない
```

その上のセクションでvarはあまり使わないようにという話をしたばかりなので、上記についても「let」で記載する方が良いかと思いますが、いかがでしょうか？

**Comment by azu** [comment]
ありがとうございます。
最初に書いたときに`var`で書いた名残りが残ってる感じがしますね。
https://github.com/asciidwango/js-primer/issues/15
https://github.com/asciidwango/js-primer/issues/46


`let`でも問題ない(ルール自体は同じ)ので、`let`に変更するのが良さそうですね。

**Reply by azu** [reply]
https://github.com/asciidwango/js-primer/issues/1388
Issueにしました。
PRも歓迎です
